### DISCUSSION TITLE
dscmd is asking for server when the connection string is passed to it

### DISCUSSION THREAD

**Discussion by tnhurst57** [DaxStudio/DaxStudio#1281] [General]
Here is the dialog - Docs says server not required if passing connection string. Trying to query our Power BI datasets:
dscmd csv myfile.csv --connectionstring "Data Source=powerbi://api.powerbi.com/v1.0/myorg/IBCE
_Dev;Initial Catalog=IB Targeting Dashboard" -q "EVALUATE 'Master'"
Error: You must specify a Server option

**Comment by dgosbell** [comment]
Sorry, I just noticed this was a discussion, not an issue. There were a couple of issues with dscmd that I fixed in the 3.2.1 release and this issue with using --connectionstring was one of them
